### PR TITLE
install ant design v4 and build main layout

### PR DESCRIPTION
**Pull request by danielcheng80**

### CODE REVIEW

**Review by KGWhite — CHANGES_REQUESTED** [review]
hello

**Review by danielcheng80 — COMMENTED** [review]
how old are you

**Review by KGWhite — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -2,14 +2,14 @@ import React from "react";
 import { BrowserRouter as Router, Switch } from "react-router-dom";
 import routes from "components/router/routes";
 import RouteWithSubRoutes from "components/router/RouteWithSubRoutes";
-import "./App.css";
+import "antd/dist/antd.css";
```
Comment (KGWhite) [inline src/App.js:5]: WTF

Comment (KGWhite) [inline src/App.js:5]: WTTTF

**Review by KGWhite — APPROVED** [review]

**Pull request merged**